### PR TITLE
chore(deps): upgrade @hawtio/react to 1.2.3

### PR DESCRIPTION
**Pull request by tadayosi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -71,5 +71,22 @@
...
+    "victory-create-container": "36.8.6",
+    "victory-legend": "36.8.6",
+    "victory-line": "36.8.6",
+    "victory-pie": "36.8.6",
+    "victory-scatter": "36.8.6",
+    "victory-selection-container": "36.8.6",
+    "victory-shared-events": "36.8.6",
+    "victory-tooltip": "36.8.6",
+    "victory-voronoi-container": "36.8.6",
+    "victory-zoom-container": "36.8.6"
```
Comment (phantomjinx) [inline package.json:90]: Sure it makes sense, but for information, can you just clarify why these are included?

File: packages/kubernetes-api/jest.config.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ export default {
     '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga|md)$':
       '<rootDir>/src/__mocks__/fileMock.js',
     '\\.(css|less)$': '<rootDir>/src/__mocks__/styleMock.js',
-    'react-markdown': '<rootDir>/../../node_modules/@hawtio/react/node_modules/react-markdown/react-markdown.min.js',
+    'react-markdown': '<rootDir>/../../node_modules/react-markdown/react-markdown.min.js',
     '@patternfly/react-code-editor': path.resolve(__dirname, './src/__mocks__/codeEditorMock.js'),
```
Comment (phantomjinx) [inline packages/kubernetes-api/jest.config.ts:19]: Is this directly related to updating @hawtio/react? If not, maybe just push it directly as a chore, rather than including it in this PR?

File: packages/oauth/package.json [hunk 3]
```diff
@@ -29,8 +29,8 @@
     "prepack": "yarn build && yarn replace-version"
   },
   "dependencies": {
-    "@hawtio/react": "^1.2.2",
-    "@thumbmarkjs/thumbmarkjs": "^0.14.6",
+    "@hawtio/react": "^1.2.3",
+    "@thumbmarkjs/thumbmarkjs": "^0.14.7",
```
Comment (phantomjinx) [inline packages/oauth/package.json:33]: Same as above. Is `thumbmarkjs` update required by @hawtio/react update?

File: packages/oauth/src/utils/crypto.ts [hunk 4]
```diff
@@ -2,7 +2,7 @@ import { getFingerprint } from '@thumbmarkjs/thumbmarkjs'
 
 export async function generateKey(salt: ArrayBufferView): Promise<CryptoKey> {
   const fingerprint = await getFingerprint()
-  const data = new TextEncoder().encode(fingerprint)
+  const data = new TextEncoder().encode(fingerprint as string)
```
Comment (phantomjinx) [inline packages/oauth/src/utils/crypto.ts:5]: Maybe just push this directly as a chore, rather than including it in this PR?

File: packages/online-shell/package.json [hunk 5]
```diff
@@ -50,7 +50,6 @@
     "openshift-logos-icon": "^1.7.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-markdown": "^9.0.1",
```
Comment (phantomjinx) [inline packages/online-shell/package.json:53]: Same as comment about `react-markdown` above.

**Comment by tadayosi** [comment]
The other changes are needed to make `yarn link` successful locally. They will be sent with a different pull req.
